code/modules/mob/living/carbon/human/species.dm

```diff
@@ -271,6 +271,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(DIGITIGRADE in species_traits)
 		C.Digitigrade_Leg_Swap(FALSE)
 
+	if(TRAIT_RADIMMUNE in inherent_traits)
```

Comment:
You need to move this down to https://github.com/tgstation/tgstation/blob/9aee2702e7e8d453bfd938aae0d256870fa7673c/code/modules/mob/living/carbon/human/species.dm#L311

Reply:
Yeah that will look better. I didn't even see those down there..